### PR TITLE
Re-Updating the codebase of airport-codes

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made:
- Added `Makefile`
- Added `.github/workflows/actions.yml` to trigger daily updates
- Removed obsolete files that are not relevant e.g `.datapackage`, `airport-codes.csv`
- Made minor changes to the `scripts/` folder 
- Added `requirements.txt`

cc @anuveyatsu

### CODE REVIEW

File: datapackage.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
...
...ng airport locations, other codes (IATA, local if exist) that are relevant to identification of an airport.  \nOriginal source url is http://ourairports.com/data/airports.csv  (stored in archive/data.csv)  \n\n## Preparation\n\nYou will need Python 3.6 or greater and [dataflows](https://pypi.org/project/dataflows/) library to run the script\n\nTo update the data run the process script locally:\n```bash\n# Install dataflows\npip install dataflows\n\n# Run the script\npython airport-codes-flow.py\n```\n\nSeveral steps will be done to get the final data.\n\n* merge columns \"latitude_deg\" and \"longitude_deg\" into \"coordinates\"\n* remove columns: \"id\",  \"scheduled_service\", \"home_link\", \"wikipedia_link\", \"keywords\"\n\n## Automation\n\nDaily updated 'Airport codes' datapackage could be found on the [datahub.io](http://datahub.io/):  \nhttps://datahub.io/core/airport-codes\n\n## License\n\nThe source specifies that the data can be used as is without any warranty. Given size and factual nature of the data and its source from a US company would imagine this was public domain and as such have licensed the Data Package under the Public Domain Dedication and License (PDDL).\n",
```
Comment (anuveyatsu) [inline datapackage.json:16]: @gradedSystem if we are deleting this, can you ensure we have this info in the README.md file?

File: scripts/airport-codes-flow.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from dataflows import validate, update_resource, add_metadata, dump_to_path
 
 
-def readme(fpath='README.md'):
+def readme(fpath='../README.md'):
```
Comment (anuveyatsu) [inline scripts/airport-codes-flow.py:7]: this probably didn't work and that's why `readme` property is null above. But we don't need this to be in the datapackage.json

**Comment by gradedSystem** [comment]
@anuveyatsu sure looking into it

File: scripts/airport-codes-flow.py (hunk 2 above)
Comment (gradedSystem) [reply scripts/airport-codes-flow.py]: It should work now I have tested it locally, actually we have to leave it like it was before:
`def readme(fpath='README.md'):`

File: datapackage.json (hunk 1 above)
Comment (gradedSystem) [reply datapackage.json:16]: fixed with the last commit

**Review by anuveyatsu — APPROVED** [review]

**Pull request merged**